mbbsd/mbbsd.c

```diff
@@ -1192,6 +1194,22 @@ user_login(void)
     setproctitle("%s: %s", margs, cuser.userid);
 #endif
 
+    // ensuring the contact email if lastlogin before sometime.
+#ifdef USEREC_EMAIL_IS_CONTACT
+    #ifdef ENSURE_CONTACT_EMAIL_LASTLOGIN
+    if(cuser.lastlogin < ENSURE_CONTACT_EMAIL_LASTLOGIN) {
+        more("etc/ensurecontactemail", NA);
+        pressanykey();
+
+        pwcuSetEmail(empty_email);
```

Comment:
This isn't ideal though. If the user disconnects, they will be left with empty contact email.
You could change the underlying functions to accept a boolean parameter that allows the same email to be used. However in that case it should still be re-verified.